### PR TITLE
Follow up about git as primary repository (ko)

### PR DESCRIPTION
**Pull request by riseshia**
Target is #2064, 93c64c4

Could you review this? 🙏 @ruby/www-ruby-lang-org-i18n-ko

### CODE REVIEW

File: ko/community/ruby-core/index.md [hunk 1]
```diff
@@ -11,50 +11,39 @@ lang: ko
...
 
 {% highlight sh %}
-$ svn co https://svn.ruby-lang.org/repos/ruby/branches/{{ site.svn.previous.branch }}
+$ git clone git@git.ruby-lang.org:ruby/ruby.git
 {% endhighlight %}
 
 이 명령은 `{{ site.svn.stable.branch }}`나 `{{ site.svn.previous.branch }}` 디렉터리에
 각각의 루비 개발 트리를 체크아웃합니다. 유지보수 브랜치에서 개발하는 개발자들은
 유지보수에서의 변경사항을 루비 트렁크에도 이관할 것입니다. 그래서 마츠나
 노부가 언어레벨의 수정을 한 경우를 제외하면 두 브랜치는 매우 비슷합니다.
```
Comment (yous) [inline ko/community/ruby-core/index.md]: This paragraph should be removed.

File: ko/community/ruby-core/index.md [hunk 2]
```diff
@@ -103,44 +93,36 @@ Subversion에 대한 정보는 [Subversion FAQ][3]나 [the Subversion book][4]
...
-루비의 코어 개발자들이 영어를 잘한다고 해도 일본에 살고 있으니 시차가 있습니다.
+루비의 코어 개발자들이 영어를 잘한다고 해도 많은 수가 일본에 살고 있으니 시차가 있습니다.
 영어와 동시에 진행되는 일어 개발 메일링 리스트도 있습니다. 클레임이 해결되지
 않았을 경우에도 인내심을 가지고 며칠 후에 다시 시도해주시길 바랍니다.
 
-### 코어 개발자를 위한 규칙
-{: #coding-standards}
+### 브랜치에 대한 안내
+
+루비의 소스 코드는 2019년 4월 22일까지 서브버전에서 관리되어왔습니다.
```
Comment (yous) [inline ko/community/ruby-core/index.md]:
```suggestion
루비의 소스 코드는 2019년 4월 22일까지 서브버전에서 관리되어 왔습니다.
```

File: ko/community/ruby-core/index.md [hunk 3]
```diff
@@ -69,23 +58,24 @@ Subversion에 대한 정보는 [Subversion FAQ][3]나 [the Subversion book][4]
```
Comment (yous) [inline ko/community/ruby-core/index.md]: Above this line, link `[11]` should be replaced with `[writing-patches]`. See #1952, the title is "패치 작성자를 위한 지침".

File: ko/community/ruby-core/index.md [hunk 4]
```diff
@@ -103,44 +93,36 @@ Subversion에 대한 정보는 [Subversion FAQ][3]나 [the Subversion book][4]
...
-  `add-log` 모드는 단축키 `C-x 4 a`로 열 수 있습니다.)
-* 체크인 날짜는 일본 표준시(UTC+9)여야 합니다.
-* ChangeLog에 추가할 때에는 Subversion 커밋 메시지도 같은 내용을 넣어야
-  합니다. 이 메시지는 커밋과 동시에 Ruby-CVS 메일링 리스트에 자동으로
-  공지됩니다.
-* 함수 프로토타입은 루비의 코드와 코드의 확장기능 전체에서 사용됩니다.
-* C++ 스타일의 주석(`//`)을 쓰지 마세요. 루비의 메인테이너는 표준 C의
-  여러 줄 주석(`/* .. */`)을 사용하길 권장합니다.
+그렇지만 (브랜치 유지보수 담당자가 아니라면) 이를 신경쓸 필요는 없습니다.
+브랜치들을 Git에서 체크아웃 할 수 있습니다.
```
Comment (yous) [inline ko/community/ruby-core/index.md]:
```suggestion
브랜치를 Git에서 체크아웃 할 수 있습니다.
```

File: ko/community/ruby-core/index.md [hunk 5]
```diff
@@ -103,44 +93,36 @@ Subversion에 대한 정보는 [Subversion FAQ][3]나 [the Subversion book][4]
...
+{% endhighlight %}
 
+X_X는 체크아웃 하고 싶은 브랜치 이름으로 변경하세요.
 
+브랜치를 변경하고 싶다면 [이슈 트랙커][10]에 이슈를 만들어주세요.
+다음 링크도 참고하세요.
 
+[gitrlo]: https://git.ruby-lang.org/ruby.git
 [mailing-lists]: /ko/community/mailing-lists/
-[1]: http://subversion.apache.org/
```
Comment (yous) [inline ko/community/ruby-core/index.md]: Above this line:
``` markdown
[writing-patches]: /ko/community/ruby-core/writing-patches/
```

**Comment by yous** [comment]
#2072

File: ko/community/ruby-core/index.md (hunk 1 above)
Comment (riseshia) [reply ko/community/ruby-core/index.md]: 287e7f89 fixed  🙈

File: ko/community/ruby-core/index.md (hunk 3 above)
Comment (riseshia) [reply ko/community/ruby-core/index.md]: Fixed f9293166

File: ko/community/ruby-core/index.md (hunk 5 above)
Comment (riseshia) [reply ko/community/ruby-core/index.md]: fixed f9293166

**Comment by riseshia** [comment]
Follow up #2072 with a792d02a

**Comment by riseshia** [comment]
Thanks!!

**Pull request merged**